src/components/Feed/Feed.js

```diff
@@ -3,12 +3,38 @@ import { Route } from 'react-router-dom';
 import PropTypes from 'prop-types';
 
 import './Feed.css';
+import Post from '../../components/Post/Post';
+import { user, posts } from '../../Data/mockData';
 
-// import Post from '../../components/Post/Post';
+const Feed = ({ profileIsVisible }) => {
+  const [currentUserPosts, setCurrentUserPosts] = useState([])
+  const [allPosts, setAllPosts] = useState([])
+
+  useEffect(() => {
+    setCurrentUserPosts(user.posts)
+    setAllPosts(posts)
+  }, [currentUserPosts, allPosts])
+
+  const displayPostsInFeed = (postsToDisplay) => {
+    return postsToDisplay.map(post => {
+      return (
+        <Post
+          userName={post.userName}
+          title={post.title}
+          date={post.date}
+          location={post.location}
+          description={post.description}
+          distance={post.distance}
+          imageUrl={post.image}
+        />
```

Comment:
Super clean! One thing I remembered just earlier that I think we'd wanted to add too was fields for quantities (either by weight or number), but think we can return to that idea later as that's covered in the mock data descriptions for now.